build.sbt

```diff
@@ -2,19 +2,55 @@ inThisBuild(
   List(
     scalaVersion := "2.12.3",
     organization := "org.scalameta",
-    version := "0.1-SNAPSHOT",
-    sources.in(Compile, doc) := Nil // faster publishLocal.
+    sources.in(Compile, doc) := Nil, // faster publishLocal
+    licenses := Seq(
+      "Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")
+    ),
+    homepage := Some(url("https://github.com/scalameta/language-server")),
+    developers := List(
+      Developer(
+        "olafurpg",
+        "Ólafur Páll Geirsson",
+        "olafurpg@gmail.com",
+        url("https://geirsson.com")
+      ),
+      Developer(
+        "gabro",
+        "Gabriele Petronella",
+        "gabriele@buildo.io",
+        url("https://github.com/gabro")
+      ),
+      Developer(
+        "laughedelic",
+        "Alexey Alekhin",
+        "laughedelic@gmail.com",
+        url("https://github.com/laughedelic")
+      )
+    ),
+    scmInfo in ThisBuild := Some(
+      ScmInfo(
+        url("https://github.com/scalameta/language-server"),
+        s"scm:git:git@github.com:scalameta/language-server.git"
+      )
+    ),
+    releaseEarlyWith := BintrayPublisher,
+    releaseEarlyEnableSyncToMaven := false,
+    releaseEarlyEnableLocalReleases := true, // TODO: remove once CI publishing is setup
+    publishMavenStyle := true,
+    bintrayOrganization := Some("scalameta"),
+    bintrayReleaseOnPublish := !isSnapshot.value
```

Comment:
@jvican I think this setting doesn't work as intended:

Reply:
The warning comes from sbt-bintray, nothing I can do about it 👍

Reply:
The warning is good! 😄 With that setting I wanted to turn off `bintrayRelease` for snapshot versions which in sbt-dynver terminology are any versions except tags (so I just realised that I actually wanted to use `isVersionStable` here). And the warning says that `bintrayRelease` is not triggered automatically on publish (which is redefined so in sbt-bintray), but then `bintrayRelease` is still done and I suspect that sbt-release-early doesn't respect this setting.